### PR TITLE
[RU] Bump all patch versions

### PR DESCRIPTION
**Pull request by brain-geek**
Hey, I've looked through not up-to-date lessons and fixed a bunch of them.

Feel free to review, @elixirschool/russian  :)

### CODE REVIEW

File: ru/lessons/basics/pattern-matching.md [hunk 1]
```diff
@@ -96,4 +96,8 @@ iex> greet.("Hello", "Sean")
 "Hi Sean"
 iex> greet.("Mornin'", "Sean")
 "Mornin', Sean"
+iex> greeting
+"Hello"
 ```
+
+Заметьте, что в примере с `"Mornin'"` переназначение переменной `greeting` в `"Mornin'"` происходит только в контексте функциии. Вне её значение `greeting` всё так же `"Hello"`.
```
Comment (nscyclone) [inline ru/lessons/basics/pattern-matching.md]: `функциии` → `функции`
Also it may be worth to change `Вне её` to something like `За её пределами`.

**Comment by eksperimental** [comment]
I dont have access to a computer now soni cannot see the exact changes of the PR. but we shouldnt bump translation versions. We update to the English version. The only ones being bumped are the English versions when they introduce a change. If you just meant to update to latest version then commit and pr message should be corrected accordingly. Thankyou

**Comment by brain-geek** [comment]
@nscyclone Applied, please look once again. 

Thanks!

**Comment by brain-geek** [comment]
Thanks the review, @nscyclone ! Merged.

**Pull request merged**